.travis.yml

```diff
@@ -5,6 +5,7 @@ sudo: false
 node_js:
   - "0.10"
   - "0.12"
+  - "4.2"
```

Comment:
I know this is merged already, but maybe we should strive to always have a test for the latest major version (5 at this time)?